test/modules/aws-ecr-integration.ts

```diff
@@ -203,7 +203,14 @@ describe('ECR Integration Testing', () => {
     `,
         (res: any[]) => {
           expect(res.length).toBe(3);
+          expect(res.map(i => i['size_in_mb']).reduce((a, b) => a + b, 0)).toBe(0);
           expect(res.filter(i => i['image_tag'] === '<untagged>').length).toBe(1);
+          const now = new Date();
+          const pushedAt = new Date(res[0]['pushed_at']);
+          expect(pushedAt.getDate()).toBe(now.getDate());
```

Comment:
isn't there a way to just do it on single line?

Reply:
It is on the same date, but at a different time. The only way I can think of is to get both dates as an ISO string and truncate them to exclude the time, but that would be harder to follow even if it involves less LOC. I could check that `now` is bigger than `pushed_at` in a single line, but that wouldn't be as exhaustive.